### ISSUE TITLE
[Feature Request] Document maxConns and maxConnLifetime

### ISSUE THREAD

**Issue by emanuelef** [temporalio/helm-charts#368]
Hi, on the helm chart I see that for sql there are maxConns and maxConnLifetime. Are they still used ?
https://github.com/temporalio/helm-charts/blob/f2992204048ace22f8bfb5ca357193f62e991829/values.yaml#L122

**Comment by robholland** [comment]
Yes, they are used. There should be nothing in values.yaml that is unused. I believe the naming is reasonably self documenting, but please re-open this issue if there is something unclear you feel that a comment would improve.
